pom.xml

```diff
@@ -43,18 +47,105 @@
     <module>hive-metastore-events</module>
   </modules>
 
+  <distributionManagement>
+    <snapshotRepository>
+      <id>egdl-maven-snapshot-local</id>
+      <url>https://artylab.expedia.biz/egdl-maven-snapshot-local</url>
+    </snapshotRepository>
+    <repository>
+      <id>egdl-maven-release-local</id>
+      <url>https://artylab.expedia.biz/egdl-maven-release-local</url>
+    </repository>
+  </distributionManagement>
+
+
   <dependencyManagement>
     <dependencies>
+<!--      Hadoop Dependencies -->
+      <dependency>
+        <groupId>org.apache.hadoop</groupId>
+        <artifactId>hadoop-common</artifactId>
+        <version>${hadoop.version}</version>
+        <exclusions>
+          <exclusion>
+            <groupId>org.apache.logging.log4j</groupId>
+            <artifactId>*</artifactId>
+          </exclusion>
+          <exclusion>
+            <groupId>log4j</groupId>
+            <artifactId>log4j</artifactId>
+          </exclusion>
+        </exclusions>
```

Comment:
Add logging exclusions to nearly every hive / hadoop package to remove the slf4j bind warnings